[6.0.0] Removed the additional word "guides" (#3555)
wso2/docs-is: en/docs/guides/adaptive-auth/adaptive-auth-overview.md

# Prerquisites for adaptive authentication

> This section provides you the prerequisites to try out the adaptive authentication scenarios configurable on Identity Server.

Reviewer: ```suggestion
This section provides the prerequisites for trying out the adaptive authentication scenarios configurable on Identity Server.
```
